admin_manual/installation/php_configuration.rst

```diff
@@ -42,13 +42,13 @@ Database connectors (pick the one for your database):
 
 *Recommended* packages:
 
-* PHP module bz2 (recommended, required for extraction of apps)
 * PHP module intl (increases language translation performance and fixes sorting
   of non-ASCII characters)
 * PHP module sodium (for Argon2 for password hashing. bcrypt is used as fallback, but if passwords were hashed with Argon2 already and the module is missing, your users can't log in.)
 
 Required for specific apps:
 
+* PHP module bz2 (for apps packaged as bz2 archive)
```

Comment:
```suggestion
```

Reply:
I think we can safely remove it outright from the docs since { core + no shipped apps need it, no currently published apps for >=NC26 are packaged as bz2}, but I have no problem leaving it as you are suggesting as an interim path. :shrug: 

Your approach is consistent with how we're treating the checks for the other modules in that section (i.e. we don't check in the core setup checks and leave it to individual apps that need one of these to check for it themselves)